docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+bot:
+  build: .
+  volumes:
+    - .:/web
+  ports:
+    - "8800:80"
+  command: /bin/bash -c "cd /web/;
```

Comment:
How often do package requirements change? For my point of view I see no sense to do a `pip install` on every container run.